lib/govuk_publishing_components/presenters/attachment.rb

```diff
@@ -1,13 +1,24 @@
 module GovukPublishingComponents
   module Presenters
     class Attachment
+      # DfE, DWP, DHSC, HMRC, DVSA and PHE are taking part in a pilot to use a form
+      # rather than direct email for users to request accessible formats. When the pilot
+      # scheme is rolled out further this can be removed.
+      EMAILS_IN_ACCESSIBLE_FORMAT_REQUEST_PILOT = %w[alternative.formats@education.gov.uk
```

Comment:
Just checked this list and it looks like some of these emails are used by multiple organisations. Firstly, PHE has been replaced by OHID and UKHSA so this is probably fine (might be worth updating the comment above to reflect this). 

However, Teaching Regulation Agency and Child Safeguarding Practice Review Panel both share DFE's email address (alternative.formats@education.gov.uk), so this change will also add these two to the pilot. Not sure how how this can be avoided with the current approach 🤔 Maybe it's okay and we just need to bubble this up to product? Otherwise there will be more work associating the accessible format email address to the organisations `content_id` which isn't currently available from the content item.

```
irb(main):023:0> pp Organisation.where(alternative_format_contact_email: EMAILS_IN_ACCESSIBLE_FORMAT_REQUEST_PILOT, govuk_status: "live").pluck(:name, :acronym, :alternative_format_contact_email)

[["Department for Education", "DfE", "alternative.formats@education.gov.uk "],
 ["Department for Work and Pensions", "DWP", "accessible.formats@dwp.gov.uk"],
 ["Department of Health and Social Care", "DHSC", "publications@dhsc.gov.uk"],
 ["HM Revenue & Customs", "HMRC", "different.format@hmrc.gov.uk"],
 ["Driver and Vehicle Standards Agency", "DVSA", "gov.uk.publishing@dvsa.gov.uk"],
 ["Teaching Regulation Agency", "TRA", "alternative.formats@education.gov.uk"],
 ["Child Safeguarding Practice Review Panel", "", "alternative.formats@education.gov.uk"],
 ["UK Health Security Agency", "UKHSA", "publications@phe.gov.uk"],
 ["Office for Health Improvement and Disparities", "OHID", "publications@dhsc.gov.uk"]]
```

Reply:
I think/hope that should be OK as the live implementation will be sending any requests for the Teaching Regulation Agency, Child Safeguarding and Department of Education to the same email box anyway. If we split it out it means that mail box would start receiving a mix of old and new style format requests. But as you say one for product to confirm.

Reply:
We have checked with the product team and confirmed that this is not a concern